src/base/pagination/test/clickEvent.test.ts

```diff
@@ -0,0 +1,40 @@
+import { elementUpdated, expect, fixture } from "@open-wc/testing";
+import { BasePagination } from "../index";
+
+describe("BasePagination", () => {
+  describe("Click events", () => {
+    it("should trigger click event when clicking on Prev button", (done) => {
```

Comment:
in my opinion , ```done``` is unnecessary 
you can use asyn/await
ref: https://github.com/kintone-labs/kintone-ui-component/blob/master/src/base/datetime/listbox/test/clickEvent.test.ts#L6